modules/crm_core_match/src/Entity/Matcher.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\crm_core_match\Entity\Matcher.
+ */
+
+namespace Drupal\crm_core_match\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+use Drupal\crm_core_contact\ContactInterface;
+use Drupal\crm_core_match\Matcher\MatcherConfigInterface;
+
+/**
+ * CRM Matcher Entity Class.
+ *
+ * @ConfigEntityType(
+ *   id = "crm_core_match",
+ *   label = @Translation("Matcher"),
+ *   config_prefix = "matcher",
+ *   handlers = {
+ *     "list_builder" = "Drupal\crm_core_match\Matcher\MatcherListBuilder",
+ *     "form" = {
+ *       "default" = "Drupal\crm_core_match\Form\MatcherForm",
```

Comment:
Core isn't really consistent about this, **NodeType** uses the same add/edit, **Vocabulary** uses default only, **CommentType** uses default/add/edit for the same form. For now, I removed default.